SoftLayer/utils.py

```diff
@@ -459,3 +461,74 @@ def decode_stacked(document, pos=0, decoder=JSONDecoder()):
         obj, pos = decoder.raw_decode(document, pos)
 
         yield obj
+
+
+def console_color_themes(theme):
```

Comment:
These functions might be better in the Environment file, I think that will be the only place that uses them (with the changes I made in https://github.com/edsonarios/softlayer-python/pull/3/files)